llvm/test/CodeGen/AArch64/GlobalISel/legalize-atan2.mir

```diff
@@ -0,0 +1,255 @@
+# RUN: llc -verify-machineinstrs -mtriple aarch64--- \
```

Comment:
Why does this file not start with:
; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
?
Look at other legalize-foo.mir files for prior art in the same directory.

Reply:
```
> ./llvm/utils/update_mir_test_checks.py llvm/test/CodeGen/AArch64/GlobalISel/legalize-atan2.mir
```

Reply:
Before, this test was manually adapted from the pattern used for some of the other arc/hyperbolic intrinsics (such as legalize-atan.mir).  I updated the test to the result of using update_mir_test_checks.py instead.

Reply:
Thanks. Automatism is great and the header gives us confidence that the check lines are correct.